### PR TITLE
[test] start add command parameters for `palette` command

### PR DESCRIPTION
**Pull request by MarekLew**

### CODE REVIEW

File: extension.js [hunk 1]
```diff
@@ -133,7 +133,8 @@ function activate(context) {
 											exec(`npm run ${command.command}`);
 										} if (command.commandType === 'palette') {
 											const executeNext = function executeNext(palette, index) {
-												vscode.commands.executeCommand(palettes[index]).then(() => {
+												const commargs = palettes[index].split(':');
```
[the remark this answers is not in the document]
Comment (MarekLew) [reply extension.js]: > Does it make sense to add a separate field "arguments" into command configuration as an Array type.

Read line bellow. I added a list of parameters, not a list parameter. 
`...` -- extracting `Array` to the parameters of `executeCommand`. `executeCommand` run function for first parameters and other parameters this function.

[the remark this answers is not in the document]
Comment (MarekLew) [reply extension.js]: Only string parameters supported.

[the remark this answers is not in the document]
Comment (MarekLew) [reply extension.js]: Chyba nie zrozumiałem twojego pytania.

> Does it make sense to add a separate field "arguments" into command configuration as an Array type?

If you change the policy for creating multiple commands(from string to array of {arrays or object}). But this breaking change.

[the remark this answers is not in the document]
Comment (ppatotski) [reply extension.js]: I was thinking about introducing a new field into configuration like so:
```json
{
	"commands": [
		{
			"text": "Deploy Commandbar",
			"color": "lightgreen",
			"command": "vsce publish",
                        "commandArguments": [
                              ["argumentforfirstcommand", "anotherargument"],
                              ["argumentforsecondcommand", 1234]
                        ]
		}
	]
}
```

[the remark this answers is not in the document]
Comment (MarekLew) [reply extension.js]: "command":[{"command1":["arg1","arg2"]},"command2",{"command3":"arg"}]

[the remark this answers is not in the document]
Comment (ppatotski) [reply extension.js]: This way is much less backward compatible, even though it is technically nice. Are you not comfortable with having "commandArguments"?

[the remark this answers is not in the document]
Comment (MarekLew) [reply extension.js]: Separate commands from arguments and count what is where? Does anyone do this? I don't understens why?

[the remark this answers is not in the document]
Comment (ppatotski) [reply extension.js]: Fine. Just change target branch to 'integration' and I will merge the code.

**Comment by ppatotski** [comment]
It is not pointed to integration branch. However I will add changes to support arguments

**Comment by MarekLew** [comment]
No problem. But fix documentation in master branch (coma '|' and colon":"), integration branch have is correct doc.

**Pull request closed without merging**